drivers/dma/dma_intel_adsp_hda_host_in.c

```diff
@@ -29,7 +29,10 @@ static const struct dma_driver_api intel_adsp_hda_dma_host_in_api = {
 												   \
 	static struct intel_adsp_hda_dma_data intel_adsp_hda_dma##inst##_data = {};                \
 												   \
-	DEVICE_DT_INST_DEFINE(inst, &intel_adsp_hda_dma_init, NULL,                                \
+	PM_DEVICE_DT_INST_DEFINE(inst, intel_adsp_hda_dma_pm_action);				   \
```

Comment:
this will compile without CONFIG_PM_DEVICE_RUNTIME?

Reply:
Same finding in code for gpdma. https://github.com/zephyrproject-rtos/zephyr/pull/52734#discussion_r1048613892